test/functional/client_side_encryption_prose_tests.js

```diff
@@ -0,0 +1,974 @@
+'use strict';
+
+const mongodb = require('../../index');
+const chai = require('chai');
+const expect = chai.expect;
+chai.use(require('chai-subset'));
+
+// Tests for the ClientEncryption type are not included as part of the YAML tests.
+
+// In the prose tests LOCAL_MASTERKEY refers to the following base64:
+
+// .. code:: javascript
+
+//   Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk
+
+describe(
+  'Client Side Encryption Prose Tests',
+  { requires: { clientSideEncryption: true } },
```

Comment:
nice! did you add a filter? I don't see it here. I was wondering if we could do this for the other test files as well. 

As an aside related to a comment in your previous PR: should this file be renamed to `test/functional/client_side_encryption/prose_tests.js`?

Reply:
uh oh. I definitely had one, but it has disappeared. I will rewrite it and submit it.